src/main/java/org/sagebionetworks/web/client/widget/entity/controller/SynapseAlertImpl.java

```diff
@@ -88,7 +89,7 @@ public void handleException(Throwable ex) {
 			view.showError(DisplayConstants.ERROR_CONFLICTING_UPDATE + "\n" + message);
 		} else if (ex instanceof DeprecatedServiceException) {
 			view.showError(DisplayConstants.ERROR_DEPRECATED_SERVICE + "\n" + message);
-		} else if (ex instanceof UnknownErrorException) {
+		} else if (ex instanceof UnknownErrorException || ex instanceof WebClientConfigurationException) {
```

Comment:
This seems appropriate.  It's an unexpected exception that we should go directly to development.